src/fsharp/FSComp.txt

```diff
@@ -361,9 +361,16 @@ csCtorHasNoArgumentOrReturnProperty,"The object constructor '%s' has no argument
 508,csNoMemberTakesTheseArguments3,"No %s member or object constructor named '%s' takes %d arguments. The named argument '%s' doesn't correspond to any argument or settable return property for any overload."
 509,csMethodNotFound,"Method or object constructor '%s' not found"
 csNoOverloadsFound,"No overloads match for method '%s'."
+csNoOverloadsFoundArgumentsPrefixSingular,"Known type of argument: %s"
+csNoOverloadsFoundArgumentsPrefixPlural,"Known types of arguments: %s"
+csNoOverloadsFoundTypeParametersPrefixSingular,"Known type parameter: %s"
```

Comment:
I'm not clear if the terminology is right, this is used to display info from `TraitConstraintInfo.ArgTys`

Reply:
let's address this in separate PR if needed